app/src/androidTest/java/ch/epfl/qedit/answer/MatrixFragmentTest.java

```diff
@@ -37,7 +37,7 @@ public void init() {
         DatabaseFactory.setInstance(dbService);
 
         Bundle bundle = new Bundle();
-        bundle.putSerializable(MatrixFragment.MATRIXID, MatrixFormat.createMatrix3x3());
+        bundle.putSerializable(MatrixFragment.MATRIX_ID, MatrixFormat.createMatrix3x3());
```

Comment:
In the PR of Alexis and myself we have put a static constant ANSWER_FORMAT which is a bit more general. I'll probably replace MATRIX_ID by this constant when merging.

Reply:
No problem, you can replace it!